source/css/_mixins/base.styl

```diff
@@ -76,11 +76,6 @@ show() {
   display: block;
 }
 
-fixbutton() {
-  right: 20px;
-  opacity: 0.8;
-}
```

Comment:
Toggle buttons (b2t too) have no `opacity` and `right` parameter. No need to remove this button.

Reply:
Then you will find b2t and sidebar button have different opacity and right-margin

Reply:
Are u sure in this?

Reply:
![image](https://user-images.githubusercontent.com/16944225/55566935-f5f91580-56fc-11e9-8d42-0cf84cf81b7b.png)

Reply:
![image](https://user-images.githubusercontent.com/16944225/55567005-1aed8880-56fd-11e9-87e7-a977fb87bb26.png)

This is your fix in this PR.